Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/audit/transform/DocSubmissionAuditTransformsTest.java

```diff
@@ -98,30 +95,26 @@ protected String getWebServiceUrlFromRemoteObject(NhinTargetSystemType target, S
 
         AssertionType assertion = createAssertion();
         LogEventRequestType auditRequest = transforms.transformRequestToAuditMsg(request, assertion,
-            createNhinTarget(), NhincConstants.AUDIT_LOG_INBOUND_DIRECTION, NhincConstants.AUDIT_LOG_ENTITY_INTERFACE,
-            Boolean.TRUE, webContextProperties, NhincConstants.NHINC_XDR_SERVICE_NAME);
+            createNhinTarget(), NhincConstants.AUDIT_LOG_INBOUND_DIRECTION, NhincConstants.AUDIT_LOG_NHIN_INTERFACE,
```

Comment:
we are not doing any check for the values passed in for transformRequestToAuditMsg() method here. These needs to be addressed in future. 
In the above line the direction passed is wrong for DS Initiator Request outbound message. Can you fix that?